councilmatic/settings_deployment.py.example

```diff
@@ -7,8 +7,8 @@ SECRET_KEY = 'replacethiswithsomethingsecret'
 TIME_ZONE = 'US/Eastern'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-# Set this to True while you are developing
-DEBUG = False
+# Set this to False while you are in production
+DEBUG = True
```

Comment:
I'm reluctant to have this True by default. let's add a line for setting DEBUG in settings_deployment in the `create your settings file` part of the readme